druid-shell/src/platform/x11/window.rs

```diff
@@ -634,22 +568,17 @@ impl Window {
             wheel_delta: delta.into(),
         };
 
-        match self.handler.try_borrow_mut() {
-            Ok(mut handler) => {
-                handler.wheel(&mouse_event);
-                Ok(())
-            }
-            Err(err) => Err(Error::BorrowError(format!(
-                "Window::handle_wheel handle: {}",
-                err
-            ))),
-        }
+        self.handler
+            .try_borrow_mut()
+            .context("Window::handle_wheel")?
```

Comment:
Lets keep the variable name in the context message. (`handler` instead of `handle` though)